assets/styles/parts/utils/_colors.scss

```diff
@@ -6,4 +6,8 @@ All variables are available here:
 https://github.com/infinum/eightshift-frontend-libs/blob/develop/styles/scss/_colors.scss
 */
 
-$base-primary-color: #FF86D6;
\ No newline at end of file
+$base-primary-color: #FF86D6;
+
+$base-text-muted-color: #999999;
```

Comment:
color should not represent the actual usage. You can use this color on divider for example and this wont make sense. Please change this